uber/automated_emails.py

```diff
@@ -131,7 +131,7 @@
 AutomatedEmail(Attendee, '{EVENT_NAME} Badge Confirmation', 'placeholders/regular.txt',
                lambda a: a.placeholder and a.first_name and a.last_name
                                        and (c.AT_THE_CON or a.badge_type not in [c.GUEST_BADGE, c.STAFF_BADGE]
-                                       and a.ribbon not in [c.DEALER_RIBBON, c.PANELIST_RIBBON, c.VOLUNTEER_RIBBON]),
+                                       and not any([ribbon in a.ribbon for ribbon in [c.DEALER_RIBBON, c.PANELIST_RIBBON, c.VOLUNTEER_RIBBON]])),
```

Comment:
This and one other line are the messiest things I had to convert to using a multichoice column. Checking for multiple values being in a string is just messy.